projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts

```diff
@@ -13,6 +17,12 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
     importPath: SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
     comment: `'${CONFIGURATOR_EVENT_LISTENER} has been removed and is no longer part of the public API. Please use 'ConfiguratorRouterListener' instead`,
   },
+  // projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts
+  {
+    node: BAD_REQUEST_HANDLER,
+    importPath: SPARTACUS_CORE,
+    comment: `'${HANDLE_VOUCHER_OPERATION_ERROR} has been removed and is no longer part of the public API. Please use new methods in ${BAD_VOUCHER_REQUEST_HANDLER}`,
```

Comment:
'${HANDLE_VOUCHER_OPERATION_ERROR} was moved to `@spartacus/cart-lib/core`.

Reply:
@WeizhengSap Hey Wei, just to clarify, the method named handleVoucherOperationError doesn't exist anymore. In the new file I do not handle voucher error in general, but instead two specific cases. Should I still use your recommended comment?